### PR TITLE
FIX: Defining the app teams bot to send the app messages

### PR DESCRIPTION
**Pull request by AllanPazRibeiro**
We encountered an issue with the retrieval of App Users within the Teams app. Currently, the creation of a bot through the app is affecting the method used to retrieve the App user. The app retrieves the user based on the appId, which is also set on the bots. As a consequence, the getAppUser function is returning the first bot it encounters, leading to unexpected behavior, such as returning the Adele bot.

To fix this, this PR: 

- Updates the method used to retrieve the app user in the Teams app.
- Replaces the usage of getAppUser with getUserByUsername where the bot user is hard-coded.

This fix ensures that the correct user is retrieved, resolving the problem within the Teams app itself.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
     "requiredApiVersion": "^1.38.0",
     "iconFile": "icon.png",
     "author": {
-        "name": "Rocket.chat",
-        "homepage": "https://www.rocket.chat/",
-        "support": "support@rocket.chat"
+        "name": "Yuqing Bian",
```
Comment (AlenDavid) [inline app.json:7]: Are we sure to change the app author name, homepage and support here?

**Review by AlenDavid — COMMENTED** [review]
I want to approve, I just don't understand the app.json changes.

File: app.json (hunk 1 above)
Comment (AllanPazRibeiro) [reply app.json:7]: ops, looks like someone  forgot to update the main before :(

**Review by AlenDavid — APPROVED** [review]
🥇 here, king, you dropped this

**Pull request merged**